pyproject.toml

```diff
@@ -110,11 +110,5 @@ Documentation = "https://docs.pyansys.com"
 [tool.flit.module]
 name = "pyansys"
 
-[tool.black]
-line-length = 100
-
-[tool.isort]
-profile = "black"
-force_sort_within_sections = true
-line_length = 100
-src_paths = ["doc", "src"]
+[tool.ruff]
+line-length = 79
```

Comment:
Let's keep the original length. Also, let's add the full configuration for the different rules. Consider these lines as an example: https://github.com/ansys/pyansys-geometry/blob/c848076c129856ac7609c924af82547db424c4b8/pyproject.toml#L121-L160